From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Jan 2016 17:58:01 +0100
Subject: Unbreak -d

Don't expect an argument so we enable debug_mode correctly
---
 nspr/pr/tests/server_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nspr/pr/tests/server_test.c b/nspr/pr/tests/server_test.c
index fe2bb92..904594e 100644
--- a/nspr/pr/tests/server_test.c
+++ b/nspr/pr/tests/server_test.c
@@ -550,7 +550,7 @@ int main(int argc, char **argv)
 	Usage: test_name -d
 	*/
 	PLOptStatus os;
-	PLOptState *opt = PL_CreateOptState(argc, argv, "d:");
+	PLOptState *opt = PL_CreateOptState(argc, argv, "d");
 	while (PL_OPT_EOL != (os = PL_GetNextOpt(opt)))
     {
 		if (PL_OPT_BAD == os) continue;
